lib/wallaby/phantom.ex

```diff
@@ -50,6 +52,36 @@ defmodule Wallaby.Phantom do
     :poolboy.checkin(Wallaby.ServerPool, server)
   end
 
+  defdelegate accept_dialogs(session),                            to: Driver
+  defdelegate accept_alert(session, open_dialog_fn),              to: Driver
+  defdelegate accept_confirm(session, open_dialog_fn),            to: Driver
+  defdelegate accept_prompt(session, input_va, open_dialog_fn),   to: Driver
+  defdelegate cookies(session),                                   to: Driver
+  defdelegate current_path!(session),                             to: Driver
+  defdelegate current_url!(session),                              to: Driver
+  defdelegate dismiss_dialogs(session),                           to: Driver
+  defdelegate dismiss_confirm(session, open_dialog_fn),           to: Driver
+  defdelegate dismiss_prompt(session, open_dialog_fn),            to: Driver
+  defdelegate get_window_size(session),                           to: Driver
+  defdelegate page_title(session),                                to: Driver
+  defdelegate page_source(session),                               to: Driver
+  defdelegate set_cookies(session, key, value),                   to: Driver
+  defdelegate set_window_size(session, width, height),            to: Driver
+  defdelegate visit(session, url),                                to: Driver
+
+  defdelegate attribute(element, name),                           to: Driver
+  defdelegate click(element),                                     to: Driver
+  defdelegate clear(element),                                     to: Driver
+  defdelegate displayed(element),                                 to: Driver
+  defdelegate selected(element),                                  to: Driver
+  defdelegate set_value(element, value),                          to: Driver
+  defdelegate text(element),                                      to: Driver
+
+  defdelegate execute_script(session_or_element, script, args),   to: Driver
+  defdelegate find_elements(session_or_element, compiled_query),  to: Driver
+  defdelegate send_keys(session_or_element, keys),                to: Driver
+  defdelegate take_screenshot(session_or_element),                to: Driver
```

Comment:
all those delegates I'm unsure about... I know it's to support that the interface just says `Wallaby.Phantom` instead of `Wallaby.Phantom.Driver` - personally though I have no problem with it saying `Wallaby.Phantom.Driver` and if I can trade all those delegates against that - then personally I'm more than fine with it :)